Setup.hs

```diff
@@ -0,0 +1,14 @@
+import Distribution.Simple
+
+main :: IO ()
+main = defaultMainWithHooks simpleUserHooks
+  {  postCopy = copyLib
+  }
+
+copyLib :: :: Args -> CopyFlags -> PackageDescription -> LocalBuildInfo -> IO ()
+copyExtLib _ flags pkg_descr lbi = do
```

Comment:
An extra-strong typing is requested for `copyLib`? :smile: 

And the names doesn't match?..

Reply:
strange... somehow a variant of that compiled for me and i'm not even sure if the Setup.hs is even called with "distributionType: Simple" in the .cabal 😓 

i was just very happy that something finally compiled after months of trying & tried to cherry-pick hunks from the now running branch ..

but yes.. you are completely right - that code is garbage .. 🙈